modules/consul_cluster/scripts/install_hashitools_consul_server.sh.tpl

```diff
@@ -19,14 +19,19 @@ INSTANCE_ID=`curl -s http://169.254.169.254/latest/meta-data/instance-id`
 AVAILABILITY_ZONE=`curl -s http://169.254.169.254/latest/meta-data/placement/availability-zone`
```

Comment:
Same comment as in https://github.com/hashicorp/terraform-aws-consul-starter/pull/31/files#r517875061

Reply:
```suggestion
AVAILABILITY_ZONE=$(curl -silent http://169.254.169.254/latest/meta-data/placement/availability-zone)
```